wallets/rn_cli_wallet/Gemfile

```diff
@@ -3,5 +3,7 @@ source 'https://rubygems.org'
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
 ruby ">= 2.6.10"
 
-gem 'cocoapods', '~> 1.13'
-gem 'activesupport', '>= 6.1.7.3', '< 7.1.0'
+# Cocoapods 1.15 introduced a bug which break the build. We will remove the upper
```

Comment:
Changes generated from the RN upgrade migration guide, manually set but copy pasted as expected.